src/surfing_penguin/API/User.py

```diff
@@ -118,12 +135,16 @@ class search_user(Resource):
     def post(self):
         search_name = api.payload['username']
         if UserFunctions.get_user(search_name) is None:
-            return {'messages': "user does not exist"}
+            return error_message("User not found")
         return UserFunctions.get_user(search_name)
 
 
-@login_manager.unauthorized_handler
 @api.marshal_with(api_return_message)
-def unauthorized():
-    # TODO: Flash this message somewhere.
+def error_message(message):
+    if message == "not login":
+        return {'messages': "Please Login First"}
+    if message == "role":
+        return {'messages': "You don't have permission!"}
+    if message == "User not found":
+        return {'messages': "User not found"}
```

Comment:
I think using if/elif would be better, and remember to make the first work upper class. Aslo, where is the orignal unauthorized() method?